pkg/status/status.go

```diff
@@ -129,12 +193,14 @@ func (s *status) setStatus(condition configv1.ClusterStatusConditionType, messag
 		return
 	}
 	s.ensureClusterOperator()
-	s.setStatusCondition(condition, message)
-	s.setOperandVersion()
-	s.updateStatus()
+	previousStatus := s.clusterOperator.Status.DeepCopy()
```

Comment:
Are you sure you cannot leverage the library-go [helpers](https://github.com/openshift/library-go/blob/master/pkg/config/clusteroperator/v1helpers/status.go) for ClusterOperator here?

Reply:
The library-go helpers does not provide a tool to copy the status.